src/proof/arith_proof.cpp

```diff
@@ -656,7 +658,7 @@ Node ProofArith::toStreamRecLFSC(std::ostream& out,
 }
 
 ArithProof::ArithProof(theory::arith::TheoryArith* arith, TheoryProofEngine* pe)
-  : TheoryProof(arith, pe), d_recorder(), d_realMode(false)
+  : TheoryProof(arith, pe), d_recorder()
```

Comment:
Seems like you can remove this field from the .h file, right?

Reply:
You are correct! Nice catch! Thanks!